### PR TITLE
Improved speed of rake export task

### PR DESCRIPTION
**Pull request by lzap**
Find each should improve batch processing a bit, I can see small boost about
1-5 % in my environment. I was also thinking removing the counter which makes
export slower on slow terminals (ssh, encryption, thousands of lines). The
patch also changed the default format from html to csv, I can change that if
you don't like it.

### CODE REVIEW

**Comment by iNecas** [comment]
I'm ok with reducing the frequncy the counter is displayed, On the other hand, I'm still in favor of showing the progress of export: long running tasks without any output are not generating very nice UX

**Comment by iNecas** [comment]
I don't thing changing the default format by default here is a good idea

**Comment by adamruzicka** [comment]
@lzap Needs a rebase

**Comment by lzap** [comment]
Dropped everything but the batch processing.

File: lib/foreman_tasks/tasks/export_tasks.rake [hunk 1]
```diff
@@ -253,7 +253,7 @@ DESC
         renderer = TaskRender.new
         total = tasks.count
 
-        tasks.each_with_index do |task, count|
+        tasks.find_each do |task, count|
```
Comment (adamruzicka) [inline lib/foreman_tasks/tasks/export_tasks.rake]:
```
Gathering 106 tasks.
rake aborted!
NoMethodError: undefined method `+' for nil:NilClass
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:258:in `block (4 levels) in <top (required)>'
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:256:in `block (3 levels) in <top (required)>'
/home/aruzicka/git/foreman-tasks/lib/foreman_tasks/tasks/export_tasks.rake:250:in `block (2 levels) in <top (required)>'
/home/aruzicka/.rbenv/versions/2.3.1/bin/bundle:23:in `load'
/home/aruzicka/.rbenv/versions/2.3.1/bin/bundle:23:in `<main>'
Tasks: TOP => foreman_tasks:export_tasks
```

Apparently `find_each` yields just the items without the index. Would need something like `tasks.find_each.with_index do |task, count|`

**Comment by adamruzicka** [comment]
@lzap any chance you'd have time to revive this in the near future?

**Comment by lzap** [comment]
Rebased

**Comment by adamruzicka** [comment]
Thank you @lzap !

**Pull request merged**